README.md

```diff
@@ -38,8 +38,7 @@ docker-compose up
 
 ### locally
 
-Please have a look at the various `README` located in the `web` and `api`
-directories, as they will provide more informations regarding how to run the app locally.
+Please follow [this link](./toolbox/docs/INSTALL_LOCALLY.md).
 
 ## how to contribute ?
```

Comment:
Maybe putting it directly in a CONTRIBUTE.md ??

Reply:
CONTRIBUTE is better suited for Pull Request etiquette no? This is not a bad idea, but maybe once the project gets bigger